FEATURES.md

```diff
@@ -27,4 +27,17 @@ Use the filter option to find the desired chart quicker from the list of charts.
 ![Repository6](images/screenshot_repository6.png)
 
 If you want to install a particular chart, simply hover the pointer over the chart name and an 'Install' button will appear, as shown in the figure.
-![Repository7](images/screenshot_repository7.png)
\ No newline at end of file
+![Repository7](images/screenshot_repository7.png)
+
+# Installed Releases list
+This informs us about the release version we are currently using. You can utilize the filter option to refine your search for a specific release by employing namespaces and freetext.
```

Comment:
It's a bit misleading. You are talking about "release version", while the list is just list of releases. You also don't show good list of several releases installed, don't point to namespaces and text filter in UI.
Your description should have more structured approach: first explain the purpose of the list of multiple releases and general filtering, then explain the contains of list rows.

Try installing more releases from "repository" section, entering different namespaces in each, to get more representative picture in UI.

Reply:
Probably I'm confused with general list description totally missing and you just described the pieces of each row. Then, we need to add that general description in the beginning.

Also, we need to describe couple of special situations regarding chart version, one is when there is an version upgrade available, another is when repository is detected for that release.